multiqc/utils/config_defaults.yaml

```diff
@@ -51,7 +51,7 @@ plots_force_interactive: false
 plots_flat_numseries: 100
 num_datasets_plot_limit: 50
 collapse_tables: true
-max_table_rows: 500
+max_table_rows: 1500
```

Comment:
What does this do? I thought we were going to try to limit reports to 500 rows via Shesmu?

Reply:
Sorry, this was from testing I did before. I will change it back.